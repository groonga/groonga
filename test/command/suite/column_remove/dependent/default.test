table_create Users TABLE_PAT_KEY ShortText
column_create Users name COLUMN_SCALAR ShortText

table_create Terms TABLE_PAT_KEY ShortText \
  --default_tokenizer TokenBigram \
  --normalizer NormalizerAuto
column_create Terms users_name COLUMN_INDEX|WITH_POSITION Users name

dump

column_remove Users name

dump
