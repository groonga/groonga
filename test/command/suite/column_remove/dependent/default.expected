table_create Users TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
column_create Users name COLUMN_SCALAR ShortText
[[0,0.0,0.0],true]
table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer TokenBigram   --normalizer NormalizerAuto
[[0,0.0,0.0],true]
column_create Terms users_name COLUMN_INDEX|WITH_POSITION Users name
[[0,0.0,0.0],true]
dump
table_create Users TABLE_PAT_KEY ShortText
column_create Users name COLUMN_SCALAR ShortText

table_create Terms TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram --normalizer NormalizerAuto

column_create Terms users_name COLUMN_INDEX|WITH_POSITION Users name
column_remove Users name
[[0,0.0,0.0],true]
dump
table_create Users TABLE_PAT_KEY ShortText

table_create Terms TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram --normalizer NormalizerAuto
